specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-08-01/vaults.json

```diff
@@ -0,0 +1,2233 @@
+          "properties": {
+            "operationId": {
+              "description": "OperationId of the Resource Move Operation",
+              "type": "string",
+              "readOnly": true
+            },
+            "startTimeUtc": {
+              "description": "Start Time of the Resource Move Operation",
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true
+            },
+            "completionTimeUtc": {
+              "description": "End Time of the Resource Move Operation",
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true
+            },
+            "sourceResourceId": {
+              "description": "Source Resource of the Resource Move Operation",
+              "type": "string",
+              "readOnly": true
+            },
+            "targetResourceId": {
+              "description": "Target Resource of the Resource Move Operation",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        },
+        "moveState": {
+          "description": "The State of the Resource after the move operation",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Unknown",
+            "InProgress",
+            "PrepareFailed",
+            "CommitFailed",
+            "PrepareTimedout",
+            "CommitTimedout",
+            "MoveSucceeded",
+            "Failure",
+            "CriticalFailure",
+            "PartialSuccess"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceMoveState",
+            "modelAsString": true
+          }
+        },
+        "backupStorageVersion": {
+          "description": "Backup storage version",
+          "enum": [
+            "V1",
+            "V2",
+            "Unassigned"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "BackupStorageVersion",
+            "modelAsString": true
+          }
+        },
+        "publicNetworkAccess": {
+          "description": "property to enable or disable resource provider inbound network traffic from public clients",
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "PublicNetworkAccess",
+            "modelAsString": true
+          }
+        },
+        "monitoringSettings": {
+          "description": "Monitoring Settings of the vault",
+          "$ref": "#/definitions/MonitoringSettings"
+        },
+        "restoreSettings": {
+          "description": "Restore Settings of the vault",
+          "$ref": "#/definitions/RestoreSettings"
+        },
+        "redundancySettings": {
+          "description": "The redundancy Settings of a Vault",
+          "type": "object",
+          "properties": {
+            "standardTierStorageRedundancy": {
```

Comment:
>standardTierStorageRedundancy [](http://example.com/codeflow?start=13&length=29)

Since this is not readonly now, you should have a PUT example with this set ?

Reply:
Added example